twilight-lavalink/Cargo.toml

```diff
@@ -37,8 +38,8 @@ twilight-gateway = { default-features = false, features = ["rustls-native-roots"
 twilight-http = { default-features = false, features = ["rustls-native-roots"], path = "../twilight-http", version = "0.16.0-rc.1" }
 
 [features]
-default = ["http-support", "rustls-native-roots"]
-http-support = ["dep:percent-encoding"]
+default = ["http2", "rustls-native-roots"]
+http2 = ["hyper/http2", "hyper-util/http2"]
```

Comment:
HTTP2 as a separate feature makes little sense because it is pointless without prior knowledge, which I assume is virtually never the case. I'd add this implicitly to the TLS features.